lib/grack/server.rb

```diff
@@ -43,13 +37,11 @@ def _call(env)
       @req = Rack::Request.new(env)
 
       cmd, path, @reqfile, @rpc = match_routing
-
       return render_method_not_allowed if cmd == 'not_allowed'
       return render_not_found if !cmd
 
       @dir = get_git_dir(path)
```

Comment:
While refactoring this, I noticed, theres a bunch of the old `if !` conditions, would you like me to just change those to `unless` real quick?